plugins/module_utils/network/sonic/config/dcbx/dcbx.py

```diff
@@ -0,0 +1,440 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The sonic_dcbx class
+It is in this file where the current configuration (as dict)
+is compared to the provided configuration (as dict) and the command set
+necessary to bring the current configuration to it's desired end-state is
+created
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import re
+from copy import deepcopy
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.cfg.base import (
+    ConfigBase,
+)
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.utils import (
+    to_list,
+)
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.facts.facts import Facts
+
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.utils.utils import (
+    get_diff,
+    remove_empties,
```

Comment:
There is no "remove_empties" function in the referenced "utils" file. This import would fail if it were executed. For the purposes of this module, neither the Ansible "remove_empties" nor the similar enterprise_sonic "utils" function "remove_none" would be suitable if this module is supporting "delete all options" for an interface.

Instead of this, it would be better to import the "remove_empties_from_list" function at this line and use that on the user's input "interfaces" list.

(Please see other related comments for "remove_empties" usage in this file.)

```suggestion
    remove_empties_from_list,
```

Reply:
@kerry-meyer same as above, see remove_empties used in other features as well. 
Will try to use remove_empties_from_list, and see if it is ok.

Reply:
Done